WORKSPACE

```
@@ -13,11 +15,11 @@ http_file(
 )
 
 http_file(
-    name = "ubuntu_20.04_1.3GB",
-    sha256 = "28ccdb56450e643bad03bb7bcf7507ce3d8d90e8bf09e38f6bd9ac298a98eaad",
+    name = "ubuntu_20.05_1.3GB",
```

The old name didn't have the patch version number, so let's keep it as it was.

```suggestion
    name = "ubuntu_20.04_1.3GB",
```